system-xml.md

```diff
@@ -0,0 +1,472 @@
+# system.xml reference
+
+- [Overview](#overview)
+- [Tabs](#tabs)
+    - [Tab attribute reference](#tab-attribute-reference)
+    - [Tab node reference](#tab-node-reference)
+    - [Example: Create a new tab](#example-create-a-new-tab)
+- [Sections](#sections)
+    - [Section attribute reference](#section-attribute-reference)
+    - [Section node reference](#section-node-reference)
+    - [Example: Create a new section](#example-create-a-new-section)
+- [Groups](#groups)
+    - [Group attribute reference](#group-attribute-reference)
+    - [Group node reference](#group-node-reference)
+    - [Example: Create a new group](#example-create-a-new-group)
+- [Fields](#fields)
+    - [Field attribute reference](#field-attribute-reference)
+    - [Field node reference](#field-node-reference)
+    - [Example: Create a new field](#example-create-a-new-field)
+    - [Common Source Models](#common-source-models)
+    - [Field Validation](#field-validation)
+
+## Overview
+
+The `system.xml` file allows you to manage the Magento system configuration. Use this topic as a general reference for
+the `system.xml` file. The `system.xml` file is located under `etc/adminhtml/system.xml` in a given Magento 2 extension.
+
+The following code snippet shows the bare skeleton of the file:
+
+```xml
+<?xml version="1.0" ?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <!-- PLACE YOUR MODULE SPECIFIC CONFIGURATION HERE -->
+    </system>
+</config>
+```
+
+> **Note**  
+> If you want instant *XSD validation in your IDE, you can
+> run `bin/magento dev:urn-catalog:generate [--ide IDE] [--] <path>`.
+
+In the `system.xml` file, it is possible to define four different types of entities, which are related to each other.
+The following section describes the relationship between tabs, sections, groups, and fields. The following screenshot
+displays the Magento 2 System Configuration in the Admin backend.
+The red squares mark the different types that are defined in the `system.xml` file:
+
+![Screenshot displaying a configured section in the Admin.](img/magento2-system-configuration.png)
+
+Tabs are used to split different configuration areas semantically. Each tab can contain one or more sections, which can
```

Comment:
Same idea with bullets here, this is quite difficult to quickly parse. Perhaps even a ascii tree-hierarchy would be nice.